internal/bgp/frr/testdata/TestSingleAdvertisementInvalidPrefix.golden

```diff
@@ -11,5 +11,10 @@ router bgp 100
   neighbor 10.2.2.254 remote-as 200
   neighbor 10.2.2.254 port 179
 
+  address-family ipv4 unicast
+    neighbor 10.2.2.254 activate
+  address-family ipv6 unicast
+    neighbor 10.2.2.254 activate
```

Comment:
this isn't right to active v4 neigh from v6 AF ?

Reply:
I wasn't sure about this, but I think it's possible to advertise an ip6 family to a neighbour reachable via ipv4 (and viceversa) ?

Reply:
It would be the same as https://github.com/metallb/metallb/pull/1014/files/b88845f28a5aa8ac430c75678d8414dec856ea14#diff-1d9a33c302812585b1b2d1897f84126492129d1f9415786a73dcfa26603686c9R53 (but we need to validate it).

Reply:
I understand the config just I doubt if it will work as is i have seen some doc about how to get this to work 
```
Enable
Configure terminal
Route-Map IPV6 permit 10
Set ipv6 next-hop FE80::F816:3EFF:FE40:2985
Exit
Router bgp 1
Neighbor 2.0.0.2 remote-as 2
Address-family ipv6 unicast
Neighbor 2.0.0.2 activate
Neighbor 2.0.0.2 route-map IPV6 out
Network 2002:2::2/128
```
might be way more involved than u think

Reply:
This part is for enabling the session only (address-family without routes) to avoid the issue described here https://github.com/metallb/metallb/pull/1014#discussion_r740421183

I think we'll need to hammer our way through v6 / dual stack anyway, so I can either remove it or leave it as it is since it's not harming v4 e2e tests.